include/ck_tile/ops/gemm/pipeline/gemm_pipeline_ag_bg_cr_comp_async_default_policy.hpp

```diff
@@ -23,43 +23,71 @@ struct GemmPipelineAgBgCrCompAsyncDefaultPolicy
     {
         constexpr index_t MPerBlock = Problem::BlockGemmShape::kM;
         constexpr index_t KPerBlock = Problem::BlockGemmShape::kK;
-        constexpr index_t KPack     = GetSmemPackA<Problem>();
+        if constexpr(is_a_load_tr<Problem>)
+        {
+            // TODO: better LDS descriptor for performance
+            // This branch is reusing the logic from UniversalGemmBasePolicy::MakeALdsBlockDescriptor
```

Comment:
since I'm copying this, maybe it would be nice to factor it out into another base method